scipy/signal/windows/windows.py

```diff
@@ -1620,6 +1620,76 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(M, nbar=4, level=-30, sym=True):
+    """
+    Return the Taylor window.
+
+    The Taylor window allows for a selectable sidelobe suppression with a
+    minimum broadening. This window is commonly used in radar processing [1]_.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an
+        empty array is returned.
+    nbar : int
+        Number of nearly constant level sidelobes adjacent to the mainlobe
+    level : float
+        Desired peak sidelobe level in decibels (db) relative to the mainlobe
+    sym : bool, optional
+        When True (default), generates a symmetric window, for use in filter
+        design.
+        When False, generates a periodic window, for use in spectral analysis.
+
+    Returns
+    -------
+    out : array
+        The window, with the center value normalized to one (the value
+        one appears only if the number of samples is odd).
+
+    See Also
+    --------
+    kaiser, bartlett, blackman, hamming, hanning
+
+    References
+    -----
+    .. [1] W. Carrara, R. Goodman, and R. Majewski, "Spotlight Synthetic
+            Aperture Radar: Signal Processing Algorithms" Pages 512-513,
```

Comment:
It's possible that the extra space here and the line below (and the ones in [2]) are problematic, not sure (RST is picky sometimes)

Reply:
I think you are correct. Have dealt with this before, didn't catch that.